pkg/controller/controller.go

```diff
@@ -205,6 +213,7 @@ func (c *Controller) sync() error {
 // 7. Namespace informer / indexer
 // 8. Service informer
 func NewController(pollInterval time.Duration, dnsSuffix string, istioClient *crd.Client, k8sClient kubernetes.Interface) *Controller {
+	queue := workqueue.NewRateLimitingQueue(workqueue.DefaultControllerRateLimiter())
```

Comment:
does ExponentialBackoff rate limiter seem better compared to the default?

Reply:
Internally this is actually calling the exponential backoff starting at 5ms and ending at a 1s max. 
Each retry will be retried after 5(ms)*10^<num-failures>.